### DISCUSSION TITLE
Stuck at autorun.sh no such file while installing WBC on Navio2

### DISCUSSION THREAD

**Discussion by tom-135** [svpcom/wfb-ng#192] [Questions]
Hi, i'm trying to install svpcom wifibroadcast on navio2 and raspberry pi 4, with TP-Link TL-WN722N wifi card.
Installing the mavlink-router, I got the error "autorun.sh no such file" when i run:
`./autogen.sh && ./configure CFLAGS='-g -O2' --sysconfdir=/etc --localstatedir=/var --libdir=/usr/lib64 --prefix=/usr`

I'm following this guide https://github.com/svpcom/wifibroadcast/wiki/Navio2-Setup
Someone can help me?

**Comment by svpcom** [comment]
`autoreconf -is`

**Reply by tom-135** [reply]
it respond with 
`autoreconf: 'configure.ac' or 'configure.in' is required`
i've already installed dh-autoreconf and autoconf package

**Reply by tom-135** [reply]
For now i workaround the problem using meson instead of make
what i've done is:
1. ensure to have meson package v0.55 or above
2. build with meson with 
`cd mavlink-router`
`sudo meson setup build .`

it's correct?
